cookiecutter.json

```diff
@@ -1,14 +1,17 @@
 {
     "full_name": "John Doe",
     "email": "myemail@here.com",
-    "repo_name": "python_template",
-    "package_name": "mypython",
-    "version": "0.1.0dev",
-    "year": "2019",
+    "package_name": "myproject",
```

Comment:
I might suggest keeping `repo_name` and separating `repo_name` and `package_name` to allow the possibility for the repository name to be different than the package name.  For example, a simpler package name or for verbosity purposes.  I.e. repo `marvin_brain` and package `brain` or repo `lvmdrp` and package `drp`.   It doesn't happen often but I've had the need for it occasionally.  The default action could be to name them the same.

Reply:
Honestly I find that three names (package, repo, and pip) are two too many and it can end up being confusing. But I've added a `repo_name` and made it the same on default to `package_name`. Hopefully most people will just click enter on all the defaults ...